site/community.html

```diff
@@ -39,6 +39,18 @@ <h2>Ask Questions</h2>
           <strong>Forums</strong>&nbsp;&nbsp;&nbsp;&nbsp;<i class="orange-text fas fa-long-arrow-right"></i>
         </footer>
       </a>
+      <a href="https://join.slack.com/t/fusionauth/shared_invite/zt-6ykfov9u-MTwA1A6ptv0Gr6KcFBVG4w" class="col-xs-12 col-lg-3 gray gray-split-card">
+        <header>
+          <img src="/assets/img/stamps/forums.svg" alt="Ask Questions">
```

Comment:
We can keep this svg or see if David has any others that we could swap in here?

Reply:
I pinged David, we'll want a new SVG.